README.md

```diff
@@ -133,44 +133,3 @@ ansible-playbook -l <host/group> echotest.yaml -vv
 ```sh
 ansible-playbook -l <host/group> echotest.yaml --extra-vars "k8s_echotest_state=absent" -vv
 ```
-
```

Comment:
I think there's also a bullet in the intro of the README that should get removed too